subsys/settings/Kconfig

```diff
@@ -16,6 +16,14 @@ if SETTINGS
 module = SETTINGS
 module-str = settings
 source "subsys/logging/Kconfig.template.log_config"
+
+# This config must be compatible with
+# nrf/subsys/partition_manager/Kconfig.template.partition_size
+config PM_PARTITION_SIZE_SETTINGS_STORAGE
```

Comment:
What will happen if i introduce similar change for usage pure NVS/FCB into code?

Reply:
Then those would have to declare their own storage partitions. Which makes sense IMO, as the `settings_storage` partition is occupied by the settings subsystem.

Reply:
Ok - I have tested local patch according to your gide - It have been working well.
I think need to add similar changes for each storage. I can do that in another patch.